lib/api-base/fastApiContainer.ts

```diff
@@ -109,16 +115,60 @@ export class FastApiContainer extends Construct {
                 environment,
                 identifier: props.apiName,
                 instanceType: taskConfig.instanceType,
-                internetFacing: config.restApiConfig.internetFacing,
+                internetFacing: false,
                 loadBalancerConfig: taskConfig.loadBalancerConfig,
             },
             securityGroup,
             vpc,
+            addNlb: true
+        });
+
+        this.alb = apiCluster.alb;
+
+        const nlbVpcLink = new VpcLink(this, 'nlb-vpc-link', {
+            targets: [apiCluster.nlb]
+        });
+
+        // get the rest api
+        const restApi = RestApi.fromRestApiAttributes(this, 'RestApi', {
+            restApiId: props.restApiId,
+            rootResourceId: props.rootResourceId,
+        });
+
+        const integration = new Integration({
+            type: IntegrationType.HTTP_PROXY,
+            integrationHttpMethod: 'ANY',
+            options: {
+                connectionType: ConnectionType.VPC_LINK,
+                vpcLink: nlbVpcLink,
+                requestParameters: {
+                    'integration.request.path.proxy': 'method.request.path.proxy'
+                },
+            },
+            uri: `${apiCluster.endpointUrl}/{proxy}`,
+        });
+
+        // create the proxy
+        const resource = restApi.root.addResource('llm').addProxy({
+            defaultIntegration: integration,
+            anyMethod: true,
+            defaultMethodOptions: {
+                authorizer: props.authorizer,
+                requestParameters: {
+                    'method.request.path.proxy': true
+                }
+            }
+        });
+
+        resource.addCorsPreflight({
+            allowOrigins: Cors.ALL_ORIGINS,
+            allowHeaders: ['*'],
         });
 
         if (tokenTable) {
             tokenTable.grantReadData(apiCluster.taskRole);
         }
+
         this.endpoint = apiCluster.endpointUrl;
```

Comment:
I missed this detail- looking in the cluster definition, this is already the ALB address